styles/main.css

```diff
@@ -0,0 +1,239 @@
+:root{
+    /* defining varibles */
+    --primary-color: #ff3b30;  /*#fca5a5*/
+    --secondary-color:#fff6ea; 
+    --action: #3b82f6;
```

Comment:
didn't got what is action? Can give more precise name

Reply:
Its was the color for action buttons.. I will update it soon.

Reply:
Done with the changes.. Please re review it